app/gui/src/dashboard/hooks/backendHooks.tsx

```diff
@@ -281,21 +286,19 @@ export interface UserGroupInfoWithUsers extends UserGroupInfo {
 export function useListUserGroupsWithUsers(
   backend: Backend,
 ): readonly UserGroupInfoWithUsers[] | null {
-  const listUserGroupsQuery = useBackendQuery(backend, 'listUserGroups', [])
-  const listUsersQuery = useBackendQuery(backend, 'listUsers', [])
-  return useMemo(() => {
-    if (listUserGroupsQuery.data == null || listUsersQuery.data == null) {
-      return null
-    } else {
-      const result = listUserGroupsQuery.data.map((userGroup) => {
-        const usersInGroup: readonly User[] = listUsersQuery.data.filter((user) =>
-          user.userGroups?.includes(userGroup.id),
-        )
-        return { ...userGroup, users: usersInGroup }
-      })
-      return result
-    }
-  }, [listUserGroupsQuery.data, listUsersQuery.data])
+  const listUserGroupsQuery = useBackendQuery(backend, 'listUserGroups', EMPTY_ARRAY)
```

Comment:
FWIW, using queries like this should be discouraged as it throws away the state of the query - loading, fetching, error, etc. instead, we should fetch on components level and if we want to hide some logic behind an abstraction - introduce a hook/function that accepts data as parameters